index.html

```diff
@@ -0,0 +1,198 @@
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatibus ut pariatur ipsam quisquam magni in ullam
+        illo nihil dolorum officiis. Asperiores adipisci quod voluptate, sit explicabo ut reiciendis ducimus? Earum?
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis nam vero itaque molestiae tempora asperiores
+        delectus, repudiandae quam! Perferendis, vitae assumenda. Eum dolore accusantium molestiae et, inventore
+        doloremque odio commodi.</p>
+    </div>
+    <div class="articles2">
+      <article>
+        <h2>Monthly Web Development</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+          quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+      </article>
+
+      <article>
+        <h2>Monthly Web Development</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+          quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+      </article>
+
+      <article>
+        <h2>Monthly Web Development</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+          quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+      </article>
+      <article>
+        <h2>Monthly Web Development</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+          quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+      </article>
+
+      <article>
+        <h2>Monthly Web Development</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+          quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+      </article>
+
+      <article>
+        <h2>Monthly Web Development</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex corporis alias delectus aut aliquid repudiandae
+          quaerat ad hic magnam pariatur quasi accusamus eaque provident, rerum officia sequi, cumque autem iusto!</p>
+      </article>
+    </div>
+    <div class="seemorearticles">
+      <h3> See more on twitter </h3>
+    </div>
+    <div class="featuresmashingbook">
+      <div class="textwrap">
+        <h1> Design system</h1>
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Et enim animi, nobis quis esse nesciunt excepturi
+          nam
+          velit. Perspiciatis laborum aliquam dolorem eveniet eaque atque dolorum amet sequi debitis exercitationem!</p>
+      </div>
+    </div>
+    <div class="articles">
```

Comment:
Here we just need to change that `<div>` with a `<section>` tag and we are good.